docs/api/hardhat/hardhat-zksync-verify.md

```diff
@@ -2,12 +2,15 @@
 
 This plugin is used to verify contracts on the zkSync Era network.
 
-## Installation
+::: warning
+- Contracts must be compiled using the official `@matterlabs/hardhat-zksync-solc` plugin. 
```

Comment:
Can you just say Hardhat in these warnings instead of specifying?

Reply:
I think it should remain specified. No specific reason though.

Reply:
If you are talking about a bunch of plugins, you have to either state all their names, or be general about them. As you have put this warning in the getting started, you should definitely be more generalized about it. This avoids text going out of date if plugins are added or removed.